tests/test_model_functions.py

```diff
@@ -1,69 +1,510 @@
+        assert unordered_equations.index(gNa) < unordered_equations.index(iNa)
+
+    def test_get_equations_for_with_dummies(self, hh_model, OXMETA):
+
+        # Tests using get_equations_for without replacing dummies with sp numbers
+        # Get ordered equations
+        ina = hh_model.get_symbol_by_ontology_term(OXMETA, 'membrane_fast_sodium_current')
+        equations = hh_model.get_equations_for([ina], recurse=False, strip_units=False)
+
+        for eq in equations:
+            if eq.lhs.name == 'sodium_channel$g_Na':
+                assert isinstance(eq.rhs, sp.Dummy)
+                break
+
+    def test_get_value(self, aslanidi_model, OXMETA):
+        """ Tests Model.get_value() works correctly. """
+
+        symbol_a = aslanidi_model.get_symbol_by_ontology_term(OXMETA, "membrane_capacitance")
+        assert aslanidi_model.get_value(symbol_a) == 5e-5
+
+    #################################################################
+    # tests for get_symbol_XXX functions
+
+    def test_get_symbol_by_cmeta_id(self, basic_model):
+        """ Tests Model.get_symbol_by_cmeta_id() works correctly. """
+
+        sv11 = basic_model.get_symbol_by_cmeta_id('sv11')
         assert sv11.name == 'env_ode$sv1'
         assert sv11.units == 'mV'
 
-    def test_get_symbol_by_cmeta_id_2(self, other_model):
-        variable = other_model.get_symbol_by_cmeta_id('testcmeta')
+    def test_get_symbol_by_cmeta_id_2(self, aslanidi_model):
+        """ Tests Model.get_symbol_by_cmeta_id() works correctly. """
+
+        variable = aslanidi_model.get_symbol_by_cmeta_id('testcmeta')
         assert variable.name == 'intracellular_ion_concentrations$Na_i'
         assert variable.units == 'millimolar'
+
+    def test_get_symbol_by_name(self, basic_model):
+        """ Tests Model.get_symbol_by_name() works correctly. """
+
+        sv11 = basic_model.get_symbol_by_name('env_ode$sv1')
+        assert sv11.units == 'mV'
+
+    def test_get_symbol_by_ontology_term(self, aslanidi_model, OXMETA):
+        """ Tests Model.get_symbol_by_ontology_term() works correctly. """
+
+        symbol_a = aslanidi_model.get_symbol_by_ontology_term(OXMETA, 'membrane_capacitance')
+        assert symbol_a.name == 'membrane$Cm'
+        assert symbol_a.units == 'nanoF'
+
+    def test_get_symbols_by_rdf(self, aslanidi_model, OXMETA):
+        """ Tests Model.get_symbols_by_rdf() works correctly. """
+
+        symbol_a = aslanidi_model.get_symbols_by_rdf(('http://biomodels.net/biology-qualifiers/', 'is'),
+                                                     (OXMETA, 'membrane_voltage'))
+        assert len(symbol_a) == 1
+        assert symbol_a[0].name == 'membrane$V'
+        assert symbol_a[0].units == 'millivolt'
+
+    ######################################################################
+    # The functions listed for ontology/rdf are tested in test_rdf.py
+    #
+    # get_ontology_terms_by_symbol()
```

Comment:
Maybe remove this list, I imagine it'll go out of sync quite easily

Reply:
Overzealous use of fixtures - changed it